service/src/test/java/eu/europa/ec/fisheries/uvms/movement/service/mapper/MovementEntityToModelTest.java

```diff
@@ -250,19 +248,19 @@ public void testMapToMovementSegment() throws MovementServiceException {
 	@Test
 	public void testOrderMovementsByConnectId() throws MovementServiceException {
 		MovementHelpers movementHelpers = new MovementHelpers(movementBatchModelBean);
-		List<UUID> connectId = new ArrayList<>();
+		List<String> connectId = new ArrayList<>();
 		List<Movement> input = new ArrayList<>();
-		UUID ID;
+		String ID;
 		for(int i = 0 ; i < 20 ; i++) {
-			ID = UUID.randomUUID();
+			ID = UUID.randomUUID().toString();
 			connectId.add(ID);
 			input.add(movementHelpers.createMovement(Math.random()* 90, Math.random()* 90, ID, "ONE", Instant.now().plusMillis((long)(Math.random() * 5000))));
 		}
 		
-		Map<UUID, List<Movement>> output = MovementEntityToModelMapper.orderMovementsByConnectId(input);
+		Map<String, List<Movement>> output = MovementEntityToModelMapper.orderMovementsByConnectId(input);
 		
 		assertEquals(connectId.size(),output.keySet().size());
```

Comment:
I believe `assertEquals` works for sets, so this could be `assertEquals(connectId, output.keySet());`. If so, replace the following `for` block.